app/views/devise/registrations/new.html.erb

```diff
@@ -12,10 +12,10 @@
         <%= f.error_notification %>
 
         <div class="form-inputs">
-          <%= f.input :email,
+          <p><%= f.input :email,
                       required: true,
                       autofocus: true,
-                      input_html: { autocomplete: "email" }%>
+                      input_html: { autocomplete: "email" }%></p>
```

Comment:
Did you try to login? I see that you put a <p> tag around the email, this might break the functionality.

Reply:
at localhost it worked, but i can delete the p tags again, because it didn't work anyways as you can see

Reply:
Cool, it'll be great if we delete anything that we don't need